.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 Cargo.lock
+data/*.bin
+data/*.gsym
```

Comment:
I try to always start these ignores with `/` so it doesn't accidentally ignore something deeper, e.g `whatever/maybe/some/data/blah.bin`.

Reply:
Makes sense. Changed.